chapters/memory.txt

```diff
@@ -1925,7 +1925,7 @@ The size of the exported file may: be larger than the size requested by
 slink:VkMemoryAllocateInfo::allocationSize.
 If pname:handleType is ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_DMA_BUF_BIT_EXT,
 then the application can: query the file's actual size with
-link:man:lseek(2)[lseek(2)].
+link:http://man7.org/linux/man-pages/man2/lseek.2.html[lseek(2)].
```

Comment:
This one seems somewhat questionable to me, since man pages vary by system, so linking directly to an arbitrary version seems a bit odd.  I'm not clear the intent here was actually to try to link to man pages explicitly, so much as reference the man pages using standard notation.

Reply:
Well, it is the least arbitrary version I can get. It is link directly from `kernel.org` man-pages project.

Intent here is to fix a link. Browser says "address not recognized" for `man:`.

Reply:
reverted